src/pages/HomeParticipant.vue

```diff
@@ -25,14 +25,14 @@
           </div>
           <div
             v-if="adminInfo?.length > 1"
-            class="flex flex-row justify-center align-items-center p-float-label dropdown-container gap-4"
+            class="flex flex-row justify-center align-items-center p-float-label dropdown-container gap-4 w-full"
           >
-            <div class="assignment-select-container flex flex-row justify-content-between">
-              <div class="flex flex-column">
+            <div class="assignment-select-container flex flex-row justify-content-between justify-content-start">
+              <div class="flex flex-column align-content-start justify-content-start w-3">
                 <PvDropdown
                   v-model="selectedAdmin"
                   :options="adminInfo ?? []"
-                  option-label="name"
+                  option-label="publicName"
```

Comment:
We should somehow account for the case where publicName is undefined so that the list rendering does not throw an error when it searches for the `publicName` key but cannot find it.

Reply:
you are right, I added a v-if clause